src/components/Error/Error.scss

```diff
@@ -0,0 +1,3 @@
+.errorMeg{
+    color:red;
+}
```

Comment:
The styling is showing up, but the `<p>` element is going on a new line, below the image.  If you move the `errorMeg` class in the component on to the `<span>` instead, and then set `display: inline-flex;` here, that should solve it.
![image](https://user-images.githubusercontent.com/55474217/165186703-a4b8efa1-1858-4763-87cd-7ce79c101ecd.png)